database/migration/src/main/resources/changesets/workflow_store_type_and_version.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="workflow-store-type-and-version">
+        <addColumn tableName="WORKFLOW_STORE_ENTRY">
+            <column name="WORKFLOW_TYPE" type="VARCHAR(30)">
```

Comment:
Curious--why such a large data type for workflow_type and workflow_type_version?